crates/db_schema/src/impls/community.rs

```diff
@@ -235,14 +236,26 @@ impl CommunityFollower {
     remote_community_id: CommunityId,
   ) -> Result<bool, Error> {
     use crate::schema::community_follower::dsl::{community_follower, community_id};
-    use diesel::dsl::{exists, select};
     let conn = &mut get_conn(pool).await?;
     select(exists(
       community_follower.filter(community_id.eq(remote_community_id)),
     ))
     .get_result(conn)
     .await
   }
+
+  pub async fn is_follower(
```

Comment:
You can remove this, IMO we shouldn't be adding vote restrictions until we have proper private communities.

Reply:
Forgot to remove this, fixed.